CMakeLists.txt

```diff
@@ -33,13 +63,24 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 ")
 include_directories(
   include
   ${catkin_INCLUDE_DIRS}
-  ${OPENVR}/headers
```

Comment:
I think I already added OpenVR/headers by `include_directories("${OPENVR}/headers" "${OPENVR}/samples")` above.
Is it correct way?